README.md

```diff
@@ -334,6 +334,22 @@ Then make a cup of tea while all of those tests run. It takes a while. If you
 have more than two cores on your machine or you don't mind pwnage, you can try
 to increase the number of parallel processes used for testing.
 
+## Releasing
+
+Follow these steps to release a new version of the `amo-validator` Python package:
+
+1. Increment the `__version__` attribute at the top of
+   `./validator/__init__.py`.
+2. Commit your change to the master branch and run `git push`.
+3. Tag master with the new version number, such as `git tag 1.9.8`.
+4. Push the new tag with `git push --tags`
+5. TravisCI will build and release a new version of `amo-validator`
+   to PyPI from your tag commit.
+   [Here is an example](https://travis-ci.org/mozilla/amo-validator/builds/90333989).
+6. Copy the deployed source distribution (`.tar.gz`) and wheel distribution
+   (`.whl`) from [amo-validator on PyPI](https://pypi.python.org/pypi/amo-validator)
```

Comment:
hmm, I didn't think wheelhouse was used anymore. I've been putting wheels in the main folder since that's where TravisCI and deployment installers look for them. I think wheelhouse was a temporary hack for some projects that needed alternate wheel distros.